quiche/src/lib.rs

```diff
@@ -949,6 +953,10 @@ impl Config {
     /// application) and will allow more data to be received as the buffer is
     /// consumed by the application.
     ///
+    /// When set to zero, either explicitly or via the default, quiche will not
+    /// give any flow control to the peer, preventing if from sending any stream
```

Comment:
```suggestion
    /// give any flow control to the peer, preventing it from sending any stream
```
(and the same for all the following comments)

Reply:
derp, thanks